scripts/cli/generate_completion_scripts.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2025 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+#
+
+GIT_ROOT=$(git rev-parse --show-toplevel)
+
+(cd $GIT_ROOT &&
+  ./gradlew cli:jvmRun --args='--generate-completion=bash' -DmainClass=org.eclipse.apoapsis.ortserver.cli.OrtServerMainKt --quiet >| integrations/completions/osc-completion.bash &&
```

Comment:
Nit: Is it encouraged to use "absolute paths" to Gradle tasks like `:cli:jvmRun`, i.e. with a leading colon.